crates/iceberg/src/spec/manifest.rs

```diff
@@ -1049,7 +1049,7 @@ pub struct DataFile {
     /// order id to null. Readers must ignore sort order id for position
     /// delete files.
     #[builder(default, setter(strip_option))]
-    pub(crate) sort_order_id: Option<i32>,
+    pub sort_order_id: Option<i32>,
```

Comment:
I remembered that we don't want users to access those fields directly. 

cc @liurenjie1024 for ideas.

Reply:
Yes, I prefer to add getter methods for these fields rather accessing fields directly.